corehq/pillows/mappings/user_mapping.py

```diff
@@ -31,6 +31,7 @@
                                                 'type': 'string'}},
                            'type': 'multi_field'},
                 'location_id': {'index': 'not_analyzed', 'type': 'string'},
```

Comment:
I didn't change the property name on couch model, so it shouldn't change here, yeah?